lib/Parser/StandardChars.h

```diff
@@ -225,6 +225,8 @@ namespace UnifiedRegex
             return true;
         }
 
+        TrivialCaseMapper() {}
+        
```

Comment:
No, we have a const instance of TrivialCaseMapper. C++ requires that any const objects that are default constructed should have a default constructor- see http://stackoverflow.com/questions/7411515/why-does-c-require-a-user-provided-default-constructor-to-default-construct-a
I can add a comment